fish_prompt.fish

```diff
@@ -391,11 +389,12 @@ function __bobthefish_prompt_virtualfish -d "Display activated virtual environme
   set_color normal
 end
 
-function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
-  [ "$theme_display_ruby" = 'no' ]; and return
-  if which rvm-prompt >/dev/null 2>&1
-    set ruby_version (rvm-prompt i v g)
-  else if which rbenv >/dev/null 2>&1
+function __bobthefish_show_ruby -d 'Current Ruby (rvm/rbenv)'
+  set -l ruby_version
+  set -q theme_display_ruby_format; or set -l theme_display_ruby_format i v g
```

Comment:
since this only applies to rvm, let's make the option `theme_display_rvm_format` and move it into the `if type rvm-prompt`